packages/trpc/server/routers/viewer/bookings/get.handler.ts

```diff
@@ -571,6 +571,38 @@ export async function getBookings({
               .orderBy("AssignmentReason.createdAt", "desc")
               .limit(1)
           ).as("assignmentReason"),
+          jsonArrayFrom(
+            eb
+              .selectFrom("BookingReport")
+              .select([
+                "BookingReport.id",
+                "BookingReport.reason",
+                "BookingReport.description",
+                "BookingReport.cancelled",
+                "BookingReport.createdAt",
+                "BookingReport.reportedById",
+              ])
+              .whereRef("BookingReport.bookingId", "=", "Booking.id")
+              .orderBy("BookingReport.createdAt", "desc")
+          ).as("reports"),
+          // Check if any booking in the recurring series has been reported by the current user
```

Comment:
Why do we need this? I tested reporting a recurring booking and saw that all occurrences had a separate booking report created for them